README.md

```diff
@@ -21,3 +21,28 @@ A multiarch [tang] Docker image, based on [Alpine Linux].
 
 [Alpine Linux]: https://alpinelinux.org/
 [tang]:         https://github.com/latchset/tang
+
+
+# Quick-start
+To start a one-shot container simply run
+`docker run -it --rm -p 8080:8080/tcp padhihomelab/tang`
+where the standard server port, `8080`, will be exposed on your host machine.
+
+## Persist data
+The "quick-start command" will not persist its data (located at `/data`) beyond the lifetime of the container.  
```

Comment:
To clarify, how about adding: "located at `/data` _within the container_"

Reply:
added your suggested info